roles/rust-foundation-project-director.md

```diff
@@ -0,0 +1,24 @@
+# Role Description: Rust Foundation Project Director
+
+Please refer to [Rust Foundation Director Roles & Responsibilities](TODO) for components of the role description that apply to all Foundation Board Directors.
+
+## Goals
```

Comment:
I think we're risking making this proposal much hard to approve if we tie defining what exactly the role of the Project Directors is to how we select them. I realize that it's hard to advertise the role to others if we can't really define what the role is, but alas that's sort of the situation we find ourselves in.

I would prefer that we keep this list short and fill it with things that are extremely unlikely to change over time and leave the last line as a call for helping shape the role together with the Council and the Foundation Staff. For example:

- Represent the needs and interests of the Rust Project at large within the Rust Foundation board
- Provide additional perspective to the Rust Foundation staff on how Foundation activities may impact the Rust Project
- Work with the Rust Leadership Council and Rust Foundation staff on refining the role of Project Director for the future

Reply:
Am I right in assuming there's an implicit pushback to the "acting as a conduit of information" item implied by this comment? I'm getting that because your list verbatim included my first bullet but didn't include the second. I'm okay with that just want to be sure that's what you meant.

Reply:
Yes, I think that that particular phrasing is a bit more aspirational than we can likely commit to, and I’d in general like to focus on more concrete examples of what the role entails. So for example instead of “conduit of information”, state that the project directors meet with the staff.

Reply:
> I think we're risking making this proposal much hard to approve if we tie defining what exactly the role of the Project Directors is to how we select them. I realize that it's hard to advertise the role to others if we can't really define what the role is, but alas that's sort of the situation we find ourselves in.

I wonder if it'd be better to separate the role description into a separate PR? Maybe we should have a time limit, like if we don't have consent on the role description by next Tuesday then we pull it out and consent to the election process by itself. When we start the Setup phase, we can either point folks at the role description PR or the finished version if we've agreed on it by then.

Reply:
I do like that idea, for now I'm going to go ahead and create separate checkboxes for the two documents so we can move ahead with the procedure part right away once we're happy with the state of the document.

Reply:
Chiming in:

> Ensure communication between the Rust Project and Foundation is robust and high fidelity.

I think this is perfect. We should be giving our directors high level goals and let them come up with a strategy they and the staff feel good about. (In less official channels, we should also share our biggest hits and misses so they learn from our mistakes!)